src/components/auction/Claimer/index.tsx

```diff
@@ -110,7 +110,7 @@ const Claimer: React.FC<Props> = (props) => {
     [derivedAuctionInfo, chainId],
   )
 
-  const isLoading = isLoadingClaimInfo || !claimableBiddingToken || !claimableAuctioningToken
```

Comment:
I actually liked the isLoadingClaimInfo information, as it prevents us from flashing wrong information.

Maybe we could use:
  const isLoading = (account  && isLoadingClaimInfo )|| !claimableBiddingToken || !claimableAuctioningToken

Reply:
Done! Also added an small improvement